XamlControlsGallery/ControlPages/StandardUICommandPage.xaml

```diff
@@ -32,7 +32,7 @@
                 </Grid.RowDefinitions>
 
                 <TextBlock TextWrapping="Wrap" Margin="0,0,0,12">
-                    XamlUICommand allows the sharing of the UX associated with a command.
+                    StandardUICommand allows the sharing of the UX associated with a command.
                     In this instance we are using a StandardUICommand to quickly place 
                     the delete command in multiple controls. The XamlUICommand contains the icon, label, 
                     keyboard shortcut, and a description.
```

Comment:
In its infinite wisdom, GitHub isn't letting me add a comment where it belongs...

The context menu key doesn't bring up the menu as expected. Something in the way ContextFlyout is hooked up to the UserControl isn't working as expected. Perhaps it's not the UserControl that's receiving keyboard focus.

Reply:
Interesting, I didn't make any changes to this sample. Are you talking about the StandardUICommand sample?

Reply:
I haven't made material changes to this sample, I have added a XamlUICommand sample. We should address this as a bug to the StandardUICommand page.

Reply:
@adambarlow - There is an [internal bug](https://microsoft.visualstudio.com/defaultcollection/OS/_workitems/edit/19902038) assigned to you tracking this issue. Would be great to address that bug along with the other changes you're making.

Reply:
I did attempt to address this bug while making this change. I have added further commentary in the bug to move that issue forward. As a matter of process, let's use that bug to address that issue, and this PR to discuss the changes presented here.